### PR TITLE
Added total piping to variable piping

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
When we moved to the new content picker a few weeks back the picker responsible for piping the total variable into the calculated summary page got left out. This PR re-adds the functionality on this page type. 

### How to review 
Check you can pipe total into the calculated summary page title and all the tests pass.

### CODE REVIEW

**Comment by SamGodwin2** [comment]
[Filter unit answers on calculated summary page by their unit types (2)](https://trello.com/c/N4BcqKwL/1260-filter-unit-answers-on-calculated-summary-page-by-their-unit-types-2)

File: eq-author/src/components/ContentPickerv2/VariablePicker.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  width: 100%;
+`;
+
+const RightPositioner = styled.div`
+  float: right;
+`;
+
+const TableHeader = styled.div`
+  display: table-header-group;
+  background: #f4f5f6;
```
[the remark this answers is not in the document]
Comment (SamGodwin2) [reply eq-author/src/components/ContentPickerv2/VariablePicker.js]: It's a good point, might need to get Joe to make and maintain a theme file with all the colors used in Author. Theres a lot of instances of hard-coded colors in our codebase.

File: eq-author/src/components/ContentPickerv2/VariablePicker.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  font-size: 11px;
```
Comment (heofs) [inline eq-author/src/components/ContentPickerv2/VariablePicker.js]: Do you think this would be better as `em` instead of `px`?

Comment (SamGodwin2) [reply eq-author/src/components/ContentPickerv2/VariablePicker.js]: That was the size in the design and 11px is a little neater than 0.6875em.

**Pull request merged**